src/System.Device.Gpio.Tests/UnixDriverTests.Linux.cs

```diff
@@ -8,8 +8,8 @@ namespace System.Device.Gpio.Tests
 {
     public class UnixDriverTests : GpioControllerTestBase
     {
-        protected override GpioDriver GetTestDriver() => new UnixDriver();
+        protected override GpioDriver GetTestDriver() => UnixDriver.CreateUnixDriver();
```

Comment:
I think we actually want to change this and have two different drivers, where one is for sysfs driver and one for libgpiod driver. The one for libgpiod can simply skip tests in case the library is not installed.

Reply:
Updated